### ISSUE TITLE
[Feature Request] Extract Monaco GLSL Plugin

### ISSUE THREAD

**Issue by YunYouJun** [zeokku/glsl.app#8]
I see that glsl.app implements full-fledged glsl language highlighting.

How about extracting it as a plugin of monaco?

**Comment by Lutymane** [comment]
@YunYouJun yes I have planned to rework the language server to make it modular and pluggable

**Comment by Lutymane** [comment]
@YunYouJun how did you found out about this editor btw?

**Comment by YunYouJun** [comment]
I recently wanted to write a shader previewer, but I didn't want to reinvent a wheel, so I searched for GLSL-related projects on github and found it.
